core-bundle/src/Framework/ContaoFramework.php

```diff
@@ -84,7 +80,7 @@ public function isInitialized(): bool
     /**
      * @throws \LogicException
      */
-    public function initialize(bool $isFrontend = false): void
+    public function initialize(): void
```

Comment:
@aschempp you added this parameter in d99e339333fd56b844c1b3353809013c375b970a
I’m not 100% sure if I removed this correctly here. Can you verify if that makes sense for you?

Reply:
The argument was necessary for routing to work. We need to boot the framework to do routing, but only the complete routing can tell if we are in frontend scope. If we don't have `TL_MODE` anymore, I don't think we still need this.